python/pyarrow/_dataset.pyx

```diff
@@ -2945,6 +2999,32 @@ cdef class Scanner(_Weakrefable):
                      object columns=None, Expression filter=None,
                      int batch_size=_DEFAULT_BATCH_SIZE,
                      FragmentScanOptions fragment_scan_options=None):
+        """
+        Create Scanner from Dataset
```

Comment:
The Scanner class docstring itself has mostly the same parameters, but a more detailed explanation. Either we can refer to that docstring for more details, or we could share the content in some way to avoid duplication.

Reply:
added a reference to the class in the docstring.